### ISSUE TITLE
UI glitch on Tabby

### ISSUE THREAD

**Issue by aarshparashar** [Eugeny/tabby#5131]
Version: 1.0.167
Platform: Linux x64 5.13.0-22-generic
Plugins: none
Frontend: xterm

Prolonged UI imprint in the background of current UI even when there's a change in the app.

**Comment by Eugeny** [comment]
Please see if launching Tabby with `--disable-gpu` helps.

**Comment by H97-Git** [comment]
Hello, I think I got the same issue, here a screenshot : ![img](https://i.imgur.com/W7m94a4.png)
But mine disappear quickly with a scroll.
And yes :heavy_check_mark: `--disable-gpu` :heavy_check_mark:  solve the problem.

Version: 1.0.167
Platform: Linux x86_64 5.15.6-2-MANJARO
Frontend: xterm

**Comment by Eugeny** [comment]
Yes, definitely. I don't have a fix for it yet, but I've just added a UI toggle to triggers --disable-gpu so that the app is at least usable.

**Comment by aarshparashar** [comment]
Closing the issue....
